ingress/controllers/nginx/examples/default/rc-default.yaml

```diff
@@ -14,11 +14,18 @@ spec:
         k8s-app: nginx-ingress-lb
         name: nginx-ingress-lb
     spec:
-      terminationGracePeriodSeconds: 60      
+      terminationGracePeriodSeconds: 60
       containers:
       - image: gcr.io/google_containers/nginx-ingress-controller:0.8.2
         name: nginx-ingress-lb
         imagePullPolicy: Always
+        readinessProbe:
+          httpGet:
+            path: /healthz
+            port: 80
```

Comment:
> Are ports 80 and 10249 equivalent inside the container ?

No. 
NGINX -> 80
Go (controller) -> 10249 -> [this](https://github.com/kubernetes/contrib/blob/master/ingress/controllers/nginx/nginx/command.go#L104) checks for nginx.
(Port 10249 should be used for both probes)